src/utils/data_loading.py

```diff
@@ -67,6 +88,11 @@ def __init__(self, images_dir: str, dem_dir: str, mask_dir: str, scale: float =
     def __len__(self):
         return len(self.ids)
     
+    def _process_stats(self):
```

Comment:
This function uses the information from the ESRI json file to calculate the correct pos_weight for BCEWithLogitsLoss